src/containers/GameContainer/useGame.ts

```diff
@@ -1,14 +1,14 @@
-import { BorderGame, Game, MainHero, TimeController } from 'game';
-import { FinishLine } from 'game/FinishLine';
-import { useEffect, useMemo, useRef, useState } from 'react';
+import { Background, Game, MainHero, SafeZone, TimeController } from 'game';
+import { MouseEvent, useEffect, useMemo, useRef, useState } from 'react';
 
 import { GameStatus } from './GameContainer.types';
 
 export const useGame = (config) => {
   const canvasRef = useRef(null);
-  const [renderTime, setRenderTime] = useState('');
+  const [renderTime, setRenderTime] = useState<number | null>(null);
   const [gameStatus, setGameStatus] = useState<keyof typeof GameStatus>(GameStatus.beforeGame);
-  const { mainHero, timeController, finishLine, game, borderGame } = useMemo(() => {
+  const [score, setScore] = useState<number | null>(null);
+  const { mainHero, timeController, game, background, safeZone } = useMemo(() => {
```

Comment:
❗️
Давай отделять разные по логике блоки
```suggestion
  const [score, setScore] = useState<number | null>(null);
  
  const { mainHero, timeController, game, background, safeZone } = useMemo(() => {
```